estate/views/estate_property_views.xml

```diff
@@ -31,6 +31,10 @@
                 <sheet>
                     <field name="name" class="h1 mb-4" />
 
+                    <group>
+                        <field name="property_tag_ids" widget="many2many_tags" string="" />
```

Comment:
@hupo-odoo I tried adding nolabel="True" instead of string="" but it did not work
any ideas for a better approach?

Reply:
We usually use `nolabel="1"` btw but it should be exactly the same and it should work so I'm not sure. I'll have a better look later

Reply:
This is the result I get with string="" (note that the tags are not aligned with the name above it, idk why)
![image](https://github.com/odoo/tutorials/assets/160074741/2a7c405d-4c4f-496a-aadc-021ea696186d)

and this is the result with nolabel="1" (now tags and name are aligned, but the tags fields are compacted together)
![image](https://github.com/odoo/tutorials/assets/160074741/cba86242-b811-4616-8fba-6aee7028999f)